charts/athens-proxy/README.md

```diff
@@ -25,14 +27,101 @@ $ helm repo add athens https://gomods.github.io/athens-charts
 $ helm repo update
 ```
 
-Next, install the chart with default values to `athens` namespace:  
+Next, install the chart with default values to `athens` namespace: 
 
 ```
 $ helm install athens/athens-proxy -n athens --namespace athens
 ```
 
 This will deploy a single Athens instance in the `athens` namespace with `disk` storage enabled. Additionally, a `ClusterIP` service will be created.
 
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` | see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#scheduling |
+| annotations | object | `{}` | Extra annotations to be added to the athens pods |
+| autoscaling.apiVersionOverride | string | `""` | Overwrite the API version used for HPA, uses 'autoscaling/v2' by default. see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2/ |
+| autoscaling.behavior | object | `{}` | Defines scaling behavior for HPA |
```

Comment:
Descriptions are inconsistent. Should be either all in the form `Does X` or `Do X` but not a mix of both.

So in this example:

```suggestion
| annotations | object | `{}` | Adds extra annotations to the athens pods |
| autoscaling.apiVersionOverride | string | `""` | Overwrites the API version used for HPA, uses 'autoscaling/v2' by default. see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2/ |
| autoscaling.behavior | object | `{}` | Defines scaling behavior for HPA |
```

or

```suggestion
| annotations | object | `{}` | Add extra annotations to the athens pods |
| autoscaling.apiVersionOverride | string | `""` | Overwrite the API version used for HPA, uses 'autoscaling/v2' by default. see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2/ |
| autoscaling.behavior | object | `{}` | Define scaling behavior for HPA |
```

WDYT?

Reply:
Actually I realize this might come from an inconsistency in the flag descriptions within the code, in which case ideally they should be changed there too, and it's IMO in the scope of a docs PR as well.

And in that case, instead of making a markdown table that describes the flags, I'd just paste the `--help` in the docs? Again, not 100% sure this is what is happening though

Reply:
Yes, inconsistencies come partially from existing docs that I did not touch yet. I will review and try to unify the descriptions a bit more. Thanks for your feedback!

The Markdown table is generated, so I have no intent to adjust that.

Reply:
Updated the initial words to be always singular.